content/index.de.md

```diff
@@ -0,0 +1,248 @@
+  <p>Entwickelt von der Arbeitsgruppe für Bildung und Öffentlichkeitsarbeit. (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Zusammenfassung" class="full" %}
+{:/}
+
+Wenn Webseiten und Webwerkzeuge richtig gestaltet und programmiert sind, können Menschen mit Behinderungen sie nutzen. Derzeit werden jedoch viele Seiten und Werkzeuge mit Barrieren entwickelt, die ihre Nutzung für manche Menschen schwer oder unmöglich machen.
+
+Das Web barrierefrei zu machen dient Einzelnen, Unternehmen sowie der Gesellschaft. Internationale Webstandards legen fest, was für Barrierefreiheit erforderlich ist.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::options toc_levels="2" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Inhaltsverzeichnis" class="full" %}
+{:/}
+
+-   Das Inhaltsverzeichnis wurde automatisch erstellt.
+{:toc}
+
+<span class="box-h box-h-simple box-h-full">Verwandte Quelle</span><br>
+{% include video-link.html title="Videoeinführung in das barrierefreie Web und W3C-Standards <em>(4 Minuten)</em>" href="https://www.w3.org/WAI/videos/standards-and-benefits/" src="/content-images/wai-intro-accessibility/video-still-accessibility-intro-16-9.jpg" %}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Barrierefreiheit im Kontext {#context}
+
+<blockquote class="pull">
+  <p>Die Stärke des Webs liegt in dessen Universalität.<br>
+    Der Zugriff aller Menschen, unabhängig von Behinderungen, ist ein unabdingbarer Bestandteil.</p>
+  <footer><cite>Tim Berners-Lee, Leiter des W3C und Erfinder des World Wide Web</cite></footer>
+</blockquote>
+
+Das Web ist grundlegend so gestaltet, dass es für alle Menschen unabhängig von deren Hardware, Software, Sprache, Ort oder Fähigkeiten funktioniert. Wenn das Web dieses Ziel erreicht, ist es für Menschen mit unterschiedlichem Hör-, Bewegungs-, Seh- und Denkvermögen zugänglich.
+
+Der Einfluss von Behinderungen ist im Web anders, weil das Web Barrieren in der Kommunikation und Interaktion beseitigt, denen Menschen in der physischen Welt begegnen. Wenn Webseiten, Anwendungen, Technologien oder Werkzeuge allerdings schlecht gestaltet sind, können sie Barrieren erzeugen, die Menschen an der Nutzung des Webs hindern.
+
+**Barrierefreitheit ist essenziell für Entwickler:innen und Organisationen, die hochwertige Webseiten und Webwerkzeuge erstellen und Menschen nicht an der Nutzung ihrer Produkte und Dienstleistungen hindern wollen.**
+
+
+## Was ist das barrierefreie Web {#what}
+
+Barrierefreies Web bedeutet, dass Webseiten, Werkzeuge und Technologien so gestaltet sind, dass Menschen mit Behinderungen sie nutzen können. Genauer gesagt können Menschen
+
+- das Web wahrnehmen, verstehen, navigieren sowie mit ihm interagieren
+- zum Web beitragen
+
+Barrierefreies Web schließt alle Behinderungen ein, die den Zugang zum Web beeinträchtigen, einschließlich
+
+- kognitive,
+- geistige,
```

Comment:
@alexejrotar I notice the use of "neurologischer" in the [Unofficial german translation of WCAG 2.1](https://outline-rocks.github.io/wcag/translations/WCAG21-de/). What would be the difference between "geistige" & "neurologischer" & the two translations?

Reply:
"geistige" and "kognitive" seem to be the same (at least I could not find any distinction), while "neurologische" means "neurological". However, I could not find a use of "neurologische Behinderung", rather "neurologische Erkrankung". I assume putting "neurologische Behinderung" would be somewhat inaccurate, so it would have to be something like: "Hör-, kognitive Behinderungen, neurologische Erkrankungen, körperliche, Sprach- und Sehbehinderungen", which I find very hard to read. So I could either put "Behinderung" explicitly after every item or restructure such that "neurologische Erkrankung" is at the end (or keep the seemingly inaccurate "neurologische Behinderung").